core/controller/src/main/scala/org/apache/openwhisk/core/loadBalancer/LoadBalancer.scala

```diff
@@ -60,6 +60,14 @@ trait LoadBalancer {
   def publish(action: ExecutableWhiskActionMetaData, msg: ActivationMessage)(
     implicit transid: TransactionId): Future[Future[Either[ActivationId, WhiskActivation]]]
 
+  /**
+   * send user memory to invokers
+   *
+   * @param userMemory
+   * @param targetInvokers
+   */
+  def sendUserMemoryToInvoker(userMemoryMessage: UserMemoryMessage, targetInvoker: Int): Unit = {}
```

Comment:
How about changing the name like this?

```suggestion
  def sendChangeRequestToInvoker(userMemoryMessage: UserMemoryMessage, targetInvoker: Int): Unit = {}
```

Reply:
Updated